src/PhpFunctionsScanner.php

```diff
@@ -3,9 +3,9 @@
 namespace WP_CLI\I18n;
 
 use Gettext\Translations;
-use Gettext\Utils\PhpFunctionsScanner;
+use Gettext\Utils\PhpFunctionsScanner as GettextPhpFunctionsScanner;
 
-class WordPressFunctionsScanner extends PhpFunctionsScanner {
+class PhpFunctionsScanner extends GettextPhpFunctionsScanner {
```

Comment:
Class should be `final` and protected methods/properties should be private (unless they were already protected in the superclass).